python/svix/api.py

```diff
@@ -62,6 +62,14 @@ def to_dict(self) -> t.Dict[str, t.Any]:
         return {k: v for k, v in asdict(self).items() if v is not None}
 
 
+@dataclass
+class CreateOptions:
```

Comment:
Overall LGTM though my only concern is that we are using the same `CreateOptions` class in every method,  We used to have this with the `ListOptions` in the past but when we wanted to add an additional option to only one type of API this became an issue, more so it's a bit confusing to have a function like `logout` take `CreateOptions` as a parameter.

I know @tasn will disagree with me on having a `<FunctionName>Options` class per function that inherits from some BaseOptions class, but IMO that would be the best path forward long term here.  At the very least though we should name this something more generic like `PostOptions`.

@tasn @svix-stephanie thoughts?

Reply:
What I'd do tbh:

`PostOptions`, we can always later have it support: `t.Union[PostOptions, LogoutPostOptions]`, so it's not a biggie.

Reply:
Agreed CreateOptions could be confusing, I've updated it to PostOptions.